### PR TITLE
Various adjustments for JReleaser

### PR DESCRIPTION
**Pull request by gsmet**
It's required for publishing to Chocolatey.

### CODE REVIEW

File: devtools/cli/distribution/release-cli.sh [hunk 1]
```diff
@@ -35,9 +42,10 @@ tar -zcf ${JAVA_BINARY_DIR}.tar.gz ${JAVA_BINARY_DIR}
 popd
 
 export JRELEASER_PROJECT_VERSION=${VERSION}
-export JRELEASER_BRANCH=main
+export JRELEASER_BRANCH=${BRANCH}
+export JRELEASER_CHOCOLATEY_GITHUB_BRANCH=${BRANCH}
 
-jbang jreleaser@jreleaser full-release \
+jbang jreleaser.java full-release \
```
Comment (gastaldi) [inline devtools/cli/distribution/release-cli.sh:48]: Can you specify a tag here instead of having the main class? @maxandersen

Comment (gsmet) [reply devtools/cli/distribution/release-cli.sh:48]: Maybe.

Doc talks about branches here: https://www.jbang.dev/documentation/guide/latest/alias_catalogs.html . Maybe it also works for tags, I don't know.

Comment (gastaldi) [reply devtools/cli/distribution/release-cli.sh:48]: How about using the GAV here instead? The jreleaser.java in the jbang-catalog doesn't seem to do much TBH

Comment (gastaldi) [inline devtools/cli/distribution/release-cli.sh:48]:
```suggestion
jbang org.jreleaser:jreleaser:1.0.0-M1 full-release \
```

Works like a charm

Comment (gsmet) [reply devtools/cli/distribution/release-cli.sh]: Thanks, fixed!

**Review by gastaldi — APPROVED** [review]

**Pull request merged**